tommath.h

```diff
@@ -66,7 +66,9 @@ typedef uint32_t             mp_word;
 #elif defined(MP_64BIT)
 /* for GCC only on supported platforms */
 typedef uint64_t mp_digit;
+#if defined(__GNUC__)
 typedef unsigned long        mp_word __attribute__((mode(TI)));
+#endif
```

Comment:
What happens here? Will mp_word just be undefined if MP_UINT64 is defined and we are not using __GNUC__?

Reply:
Yes, that's it. Using tommath.h with MSVC in MP_64BIT mode makes the mp_word type unusable. But that doesn't matter since mp_word is not used in any public API: It works!

Reply:
See #267, I think that explains it all